commcare/src/main/java/org/motechproject/commcare/tasks/builder/FormTriggerBuilder.java

```diff
@@ -58,20 +58,22 @@ public FormTriggerBuilder(CommcareSchemaService schemaService, CommcareConfigSer
             for (CommcareApplicationJson application : schemaService.retrieveApplications(config.getName())) {
                 for (CommcareModuleJson module : application.getModules()) {
                     for (FormSchemaJson form : module.getFormSchemas()) {
-                        List<EventParameterRequest> parameters = new ArrayList<>();
-                        String applicationName = application.getApplicationName();
-                        String formName = form.getFormName();
-                        addMetadataFields(parameters);
-                        addCaseFields(parameters);
+                        if (form.getXmlns() != null) {
```

Comment:
Do we have this case covered by tests? If not, please add a test that verifies this.